README.md

```diff
@@ -9,8 +9,51 @@ https://github.com/psychoinformatics-de/processing-workflow.
 
 ## Software requirements for automatic recomputation
 
-- [DataLad](http://www.datalad.org)
-- [Singularity](https://sylabs.io/docs/)
+Software requirements for the worflow are described in details [here](https://github.com/psychoinformatics-de/fairly-big-processing-workflow#software-requirements).
+
+If you're not planning to use any job scheduling / batch processing system (e.g. [HTCondor](https://research.cs.wisc.edu/htcondor/), [SLURM](https://slurm.schedmd.com/documentation.html)), but would simply like to run the example below, make sure to have the following software installed:
+
+- [DataLad](http://www.datalad.org): Please make sure that you have installed a recent version of DataLad (`0.14.3` or higher), as well as recent versions of its dependencies (git, `2.24.0` or higher; git-annex, `8.20*` or higher). Installation instructions are available at: [http://handbook.datalad.org](http://handbook.datalad.org/en/latest/intro/installation.html#install). While there are many ways of installing DataLad, doing it via [Conda](http://handbook.datalad.org/en/latest/intro/installation.html#conda) is the most convenient one.
+
+- [Singularity](https://sylabs.io/docs/): In principle, no specific version of Singularity is required. If you're not sure what version to use, simply install the most recent one. Installation instructions can be found [here](https://sylabs.io/docs/).
+
+Make sure to have your [Git identity](http://handbook.datalad.org/en/latest/intro/installation.html#initial-configuration) set up.
+
+## Installation tips for MacOS
+
+It is not possible to run the workflow natively on MacOS because Singularity is currently [incompatible](https://sylabs.io/guides/3.8/admin-guide/installation.html#installation-on-windows-or-mac) with this system.
+
+Therefore, using a virtual machine is currently the only available option for MacOS users. It is recommended to use [VirtualBox](https://www.virtualbox.org) and (optionally) [Vagrant](https://www.vagrantup.com) software. This software can be installed via [Homebrew](https://brew.sh) package manager.
```

Comment:
I would probably phrase this a bit less like "your OS is not supported" and rather like "here is how you do it". Below is a potential suggestion:

```suggestion
An installation of [Singularity on MacOS(https://sylabs.io/guides/3.8/admin-guide/installation.html#installation-on-windows-or-mac) requires using a virtual machine. It is recommended to use [VirtualBox](https://www.virtualbox.org) and (optionally) [Vagrant](https://www.vagrantup.com) software. This software can be installed via [Homebrew](https://brew.sh) package manager.
```

Reply:
Sounds good. :)